module/input/NatNet/data/config/NatNet.yaml

```diff
@@ -1,3 +1,4 @@
 multicast_address: 239.255.42.99
 command_port: 1510
 data_port: 1511
+dump_packets: true
```

Comment:
I feel like it might be better to use the log level instead of having this here. Maybe log level trace will dump the packets?

Reply:
I think it should stay separate, dumping packets uses up disk space which should be a separate decision.
Should have a config option here for where to dump them though

Reply:
Oh this is saving to a file, not printing to the terminal. That's fine then.